CHANGELOG.md

```diff
@@ -1,3 +1,14 @@
+# master
+
+* The update to the version of Terraform AWS VPC module will require the following manual edits to the state file:
+
+```bash
+terraform state rm module.odc_eks.module.vpc[0].aws_vpc_endpoint_route_table_association.private_s3
+terraform state rm module.odc_eks.module.vpc[0].aws_vpc_endpoint_route_table_association.public_s3
```

Comment:
In Terraform 1.7, these edits can now be done in configuration, but we have already made this change to our state files, so we wouldn't be able to test the new way. See https://www.hashicorp.com/blog/terraform-1-7-adds-test-mocking-and-config-driven-remove.

Reply:
I have a cluster I can test this against so will let you know.

Reply:
I have tested this successfully